### ISSUE TITLE
RadSideDrawer on page level should be able to stretch at least to the bottom of the page

### ISSUE THREAD

**Issue by stefan-kolb** [NativeScript/nativescript-ui-feedback#1058]
Related to discussion in https://github.com/NativeScript/NativeScript/issues/5782

If using the side drawer only on one page not as a global root sidedrawer it does not stretch to the bottom of the page. Whereas not stretching over the action bar is visually ok, the described scenario creates an ugly visual scenario, especially on phones like iPhone X. It also seems like the layout algorithm still thinks it can use the space at the bottom (it cuts of the button in the picture).

![image](https://user-images.githubusercontent.com/2141507/54741244-2ea5d480-4bbe-11e9-82a6-2688e5a529d2.png)

__This makes it impossible to fill the background of the page.__
If the side drawer is used in the app component it also stretches to the bottom.

Using the side drawer as a root component is not always a good scenario.
Disabling the side drawer on every page (there might be quite a few) feels like a hack (and it is even pretty hacky to do so, as the side drawer is not immediately available to the page. If I'm not mistaken it was something like:

```typescript
`ngAfterViewInit() {`
    // use setTimeout otherwise there is no getRootView valid reference
    setTimeout(() => {
        const drawer = <RadSideDrawer>app.getRootView();
        drawer.gesturesEnabled = false;
    }, 100);
}
```

Additionally, if I move the side drawer to the root component, I get problems with the side drawer and `ScrollView` on pages. It will disable scrolling and hide form fields when the soft keyboard comes up. There are several issues reporting such problems without an acceptable workaround (e.g. https://github.com/telerik/nativescript-ui-feedback/issues/730).

Demo Project: https://play.nativescript.org/?template=play-ng&id=9dMwDw&v=25

**Comment by stefan-kolb** [comment]
Update from my tests: 

I was able to cover the entire background by wrapping the sidedrawer code inside a GridLayout, similar to the template inside NativescriptPlayground.

```
<ActionBar/>

<GridLayout rows="*" class="bgstyle">

<RadSideDrawer>
<GridLayout tkDrawerContent/>
<Docklayout tkMainContent />
</RadSideDrawer>

</GridLayout>
```

In the first place (or randomly) it gets rendered correctly in the emulator. however, after navigating back from another page or on a real device it renders inside the safe area somehow which may cut the contents of the main area.

![image](https://user-images.githubusercontent.com/2141507/54809037-59596100-4c82-11e9-9dc4-50d80aee5990.png)

![image](https://user-images.githubusercontent.com/2141507/54809192-de447a80-4c82-11e9-9f87-1d9a9e9359a1.png)

**Comment by tgpetrov** [comment]
Released with [nativescript-ui-sidedrawer@7.0.0](https://github.com/NativeScript/nativescript-ui-feedback/blob/master/releases/sidedrawer.md#700-2019-july-17)
